src/main/java/com/vesta/controller/UserController.java

```diff
@@ -98,6 +101,7 @@
     @PostMapping("reset/forgot/password")
     void resetForgotPassword(@RequestBody UserResetForgotView userResetForgotView);
 
+    @PreAuthorize("hasRole('USER')")
```

Comment:
This method is for all roles, it's not necessary add PreAuthorize

Reply:
I deleted this method.